### ISSUE TITLE
'Add new item in visual F# project result in the error below.

### ISSUE THREAD

**Issue by neolem** [Microsoft/visualfsharp#562]
![2015-07-30_17-42-28](https://cloud.githubusercontent.com/assets/315907/8989239/cc47e1a0-36e2-11e5-9d06-f56a8ee27622.png)

**Comment by latkin** [comment]
Hi @neolem - thanks for the issue report.  Can you provide a repro project and exact steps to make this occur?  Are you using the Visual F# Power Tools? What version of Visual Studio?

**Comment by neolem** [comment]
Hi @latkin. Yes, I am using the latest version of Visual F# Power Tools(2.0.0) which I updated just about today. So maybe the VFPT has something to do with it as I didn't experience this fault until today. I am also using the just released Visual Studio 2015. I am also using `paket` generated F# project scaffold. 

I got this error after clicking 'Add Above' or 'Add Below' menu options. 

I tried to replicate this error in another project but haven't been able to so far. I will update with more information once I can reliably repro this.
